clients/web/src/views/body/SystemConfigurationView.js

```diff
@@ -65,7 +65,10 @@ var SystemConfigurationView = View.extend({
                 this.$('#g-settings-error-message').text(resp.responseJSON.message);
             }, this));
         },
-        'click #g-restart-server': restartServerPrompt
+        'click #g-restart-server': restartServerPrompt,
+        'click #g-core-banner-default-color': function (event) {
```

Comment:
Since we're not using `event`, you can omit it. Javascript is fine with passing too many parameters (they are always available in the magic `arguments` variable) and not enough parameters (arguments default to `undefined` in the callee) into a function.

Reply:
To be more specific, only traditional `function` functions have a magic `arguments` variable. Arrow `=>` functions do not. Both types of functions can still take too many or too few parameters.